Tests/UnitTests/Examples/Algorithms/Digitization/ModuleClustersTests.cpp

```diff
@@ -46,10 +47,10 @@ DigitizedParameters makeDigitizationParameters(const Vector2 &position,
 auto testDigitizedParametersWithTwoClusters(bool merge, const Vector2 &firstHit,
                                             const Vector2 &secondHit) {
   BinUtility binUtility;
-  binUtility +=
-      BinningData(BinningOption::open, BinningValue::binX, 20, -10.0f, 10.0f);
-  binUtility +=
-      BinningData(BinningOption::open, BinningValue::binY, 20, -10.0f, 10.0f);
+  binUtility += BinUtility{
```

Comment:
`+=` 🥲

Reply:
Yeah, I don't love this either, but I'd like to not change this as well in this PR.